modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -945,6 +945,68 @@ public String getTypeDeclaration(Schema p) {
         return super.getTypeDeclaration(target);
     }
 
+    private String getBeanValidation(Schema<?> items) {
+        if (Boolean.FALSE.equals(additionalProperties.getOrDefault(BeanValidationFeatures.USE_BEANVALIDATION, Boolean.FALSE))) {
+            return "";
+        }
+
+        if (ModelUtils.isTypeObjectSchema(items)) {
+            // prevents generation '@Valid' for Object
+            return "";
+        }
+
+        // adds '@Valid' also to response...
+//        if (items.get$ref() != null) {
+//            return "@Valid ";
+//        }
+
+        if (StringUtils.isNotEmpty(items.getPattern()) && !ModelUtils.isByteArraySchema(items)) {
+            return String.format(Locale.ROOT, "@Pattern(regexp = \"%s\")", items.getPattern());
```

Comment:
It seems the code only supports one or sometimes two validations to be present simultaneously. Do you mind changing this method so that any combination if validations is supported?
e.g. a string with `pattern` and `minLength` and `maxLength`